source/Tests/Testing/TestingExtensionMethods.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace Tests.Testing
+{
+    public static class TestingExtensionMethods
+    {
+        public static string InIntegrationTestNamespace(this string classStructureToTest)
+        {
+            return $@"
+using System;
+
+namespace Octopus.IntegrationTests
+{{
+    public abstract class LoggedTest : IDisposable
+    {{
+        public void Dispose() {{ }}
+    }}
+
+    public abstract class IntegrationTest : LoggedTest {{ }}
+
+    public abstract class UnitTest : LoggedTest {{ }}
```

Comment:
Does this need to be here? Seems like a separate extension should be made for `InUnitTestNamespace`

Reply:
Good catch. I'm thinking that I'm just going to rename this to `InTestNamespace` for now.

We could _technically_ say that this is _currently_ correct, because we currently have `IntegrationTest` and `UnitTest` in the integration test project (eventually we'll have both in the same project to eliminate a bunch of excess file copying that we do). But when we do merge the two together we'll rename it to Octopus.Tests or something like that, and this current name would make even less sense.

Ideally I'd like to reference the integration test project/package from these tests (like I'm doing with the xUnit ones), but I don't believe we publish the integration test project anywhere. We'd also end up with some messy references that we'd have to sort out.

Reply:
Reading your next comment below, I think it would probably be better if this was something like "WithTestingTypes" or something to better communicate what this method is adding to the codes. We're just making the types available to the test code.